x-pack/plugins/logstash/common/constants/monitoring.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const MONITORING = {
+  PIPELINE_RECENCY_DURATION_S: 30
```

Comment:
Yeah, I agree. I struggled with naming it too 😄.

I'll add an explanatory comment for now but even better would be a self-explanatory name. If you have suggestions just post them here!

Reply:
I'll think about this some more. I was on a time crunch when I posted the initial review, so I copped out and asked for a comment 😄

Reply:
- `IGNORE_DELETED_PIPELINES_DURATION`
- `PIPELINE_DELETION_DELAY`
- `PIPELINE_DELETION_FRESHNESS`

Reply:
I decided to keep the constant name the same because its actually used in two places, deliberately and not just by coincidence. However, I did add a comment explaining the constant's meaning in d42e147.